eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -87,10 +93,44 @@ stages:
       useMonoRuntime: false              # 🚫
       withPreviousSDK: false             # 🚫
 
-  - ${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
+  - template: ../jobs/vmr-build.yml
+    parameters:
+      # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
+      buildName: CentOSStream8_Online_PreviousSourceBuiltSdk
```

Comment:
Per [this comment](https://github.com/dotnet/source-build/issues/3463#issuecomment-1699824757) the intent is for this to be a stage 2 leg which rebuilds using the SDK from the CentOSStream8_Online_MsftSdk leg.  This should be named `CentOSStream8_Online_CurrentSourceBuiltSdk` and configured accordingly.

Reply:
Addressed in [this commit](https://github.com/dotnet/installer/pull/17370/commits/aad69ec5a52c6b74d4f6e4dfd4edc2ecdf8e115b).